.github/shared/src/readme.js

```diff
@@ -160,6 +181,9 @@ export class Readme {
       }
 
       this.#data = { globalConfig, tags };
+
+      // Clear #content to save memory, since it's no longer needed after #data is loaded
+      this.#content = undefined;
```

Comment:
This initially confused me because it looks like the private member is set at the beginning of the method and deleted before the end of the method... That would be redundant except you can set the content in the constructor. 

This would mean that `getData` is the only logic which depends on `#content`... which I think is reasonable at this point.